client/src/components/helpers/declarations.d.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/unambiguous
+declare module 'react-test-renderer';
```

Comment:
Could you import `@types/react-test-renderer` instead?

I should have put this in the original issue, but I'm still figuring this stuff out.  Sorry!

Reply:
No worries. You want me to import it in place of line 2 above. Right?

Reply:
Sorry, I shouldn't have said 'import', my bad.  As I understand it, you just need to `npm -D @types/react-test-renderer` inside the `client` directory and TypeScript will look for it automatically.